src/components/Home/Aditya/Week3/utils/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const removeWhitespace = (str: string): string => {
+  let currentTask = str.replace(/\s+/g, ' ');
+  if (currentTask) {
+    if (currentTask[0] === ' ') {
```

Comment:
Why extra space is there in condition '  '

Reply:
It stops the string from having spaces at start
we can use trim(), but that will remove spaces at end which we don't want.